api/mesh/v1alpha1/dataplane.proto

```diff
@@ -97,48 +120,75 @@ message Dataplane {
         Tcp tcp = 5;
       }
 
-      // ServiceProbe defines parameters for probing service's port
+      // ServiceProbe defines parameters for probing the service next to
+      // sidecar. When service probe is defined, Envoy will periodically health
+      // check the application next to it and report the status to the control
+      // plane. On Kubernetes, Kuma deployments rely on Kubernetes probes so
+      // this is not used.
+      //
+      // See https://kuma.io/docs/latest/documentation/health for more
+      // information.
       ServiceProbe serviceProbe = 8;
     }
 
-    // Outbound describes a service consumed by the dataplane.
+    // Outbound describes a service consumed by the data plane proxy.
+    // For every defined Outbound there is a corresponding Envoy Listener.
     message Outbound {
-      // Address on which the service will be available to this dataplane.
-      // Defaults to 127.0.0.1
+      // IP on which the consumed service will be available to this data plane
+      // proxy. On Kubernetes, it's usually ClusterIP of a Service or PodIP of a
+      // Headless Service. Defaults to 127.0.0.1
       string address = 3;
 
-      // Port on which the service will be available to this dataplane.
+      // Port on which the consumed service will be available to this data plane
+      // proxy. When transparent proxying is not used, Envoy will bind to this
+      // port.
       uint32 port = 4 [ (doc.required) = true ];
 
-      // DEPRECATED: use networking.outbound[].tags
-      // Service name.
+      // DEPRECATED: use `networking.outbound[].tags['kuma.io/service']`
+      // Service name identified by the value of `kuma.io/service`.
       string service = 2
           [ (validate.rules).string.hostname = true, deprecated = true ];
```

Comment:
Do we have an issue to actually remove it?

Reply:
We talked that we want to keep it as is for now, because we may want to make service more explicit instead of a tag in which case this field would be useful.